_docs/backenders/strategy-2021/roles/capacity.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Template
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Capacity role definition
+
+**Current responsible**: -
+
+## Responsibility:
+    **Meetings and other FA:**
+- Facilitator on the capacity part of the BE SMO meeting.
+- Attend to capacity monthly meeting.
+- Participate in fortnightly BD meetings.
+- Participate in BD/FE retrospectives every 6 months.
+- Point of contact for other functional areas.
+
+    **Inside the team:**
+- Having a track on the project proposal pipe.
+- Map and communicate skills.
+- Improve communication of medium and long-term capacity estimations.
+- Listen and look for solutions to the group needs on capacity wise making sure capacity needs from the BE group are covered.
+- Push people to participate on capacity wise and ensure that, as a team we are aware and keep track of current allocations and delayed projects.
+- Improve the proposals allocation workflow and facilitate allocating people required if needed based on availability, interests and skills. 
+
+Not clear:
+Drive and promote automation for data collection and modelling predictions.
```

Comment:
@aagm Shall we refactor this sentence to make it sound less complicated and/or ambitious?

Reply:
i had pushed some changes there.